.travis.yml

```diff
@@ -18,3 +16,6 @@ deploy:
   local_dir: _book
   upload-dir: docs
   skip_cleanup: true
+  on:
+    branch: master
+    condition: "$TRAVIS_PULL_REQUEST = true"
```

Comment:
why? I'm confused.

Reply:
Well spotted. I've mixed the previous build condition with the `||` operator - it should deploy when PR is `false` 👍